### PR TITLE
Added method for reading gripper-state

### PR DESCRIPTION
**Pull request by zephirefaith**
TIL

### CODE REVIEW

**Review by exhaustin — APPROVED** [review]
Looks good!

File: src/home_robot_hw/home_robot_hw/remote/modules/manip.py [hunk 1]
```diff
@@ -78,6 +78,9 @@ def get_joint_positions(self):
             q[HelloStretchIdx.WRIST_PITCH],
             q[HelloStretchIdx.WRIST_ROLL],
         ]
+    def get_gripper_position(self):
```
Comment (cpaxton) [inline src/home_robot_hw/home_robot_hw/remote/modules/manip.py:81]: add type hints + docstring

**Review by cpaxton — CHANGES_REQUESTED** [review]
add a docstring + return type hint and fix the linter errors

**Review by cpaxton — APPROVED** [review]
lgtm

**Pull request merged**